lib/ansible/modules/monitoring/nagios.py

```diff
@@ -294,13 +293,7 @@ def main():
         # Make sure there's an actual service selected
         if not services:
             module.fail_json(msg='no service selected to set downtime for')
-        # Make sure minutes is a number
-        try:
-            m = int(minutes)
-            if not isinstance(m, types.IntType):
-                module.fail_json(msg='minutes must be a number')
-        except Exception:
-            module.fail_json(msg='invalid entry for minutes')
+        m = int(minutes)
```

Comment:
The value of `m` is never used again. I think you can simply remove this whole conversion.

Reply:
In line 285, also remove mention of `minutes`.